templates/web/base/alert/_list.html

```diff
@@ -6,88 +6,106 @@
   <input type="hidden" name="latitude" value="[% latitude | html %]">
   <input type="hidden" name="longitude" value="[% longitude | html %]">
 
-<fieldset>
- <legend>
-  <h2>[% loc('Which problems do you want alerts about?') %]</h2>
- </legend>
+  <div class="govuk-form-group">
+    <fieldset class="govuk-fieldset">
+    <legend>
+      <h2 class="govuk-fieldset__legend govuk-fieldset__legend--m">[% loc('Which problems do you want alerts about?') %]</h2>
+    </legend>
 
-  [% INCLUDE 'errors.html' %]
+      [% INCLUDE 'errors.html' %]
 
-  [% IF reported_to_options %]
-    <p>
-        [% tprintf(loc('Reports near %s are sent to different councils, depending on the type of problem.'), name_of_location) %]
-    </p>
-    <p>
-        [% loc('You can choose to subscribe to all problems reported in an area, or reports based on their destination.') %]
-    </p>
-    <div class="alerts__columns">
-        <div>
-    <h3>[% loc('Problems in an area') %]</h3>
-  [% END %]
+    <div class="govuk-radios" data-module="govuk-radios">
```

Comment:
We don't use this, so this wrapper div isn't actually needed (but no real harm leaving it in, I guess).